functions-and-operators/tidb-functions.md

```diff
@@ -17,6 +17,7 @@ The following functions are TiDB extensions, and are not present in MySQL:
 | [`TIDB_VERSION()`](#tidb_version) | The `TIDB_VERSION` function returns the TiDB version with additional build information. |
 | [`TIDB_DECODE_SQL_DIGESTS(digests, stmtTruncateLength)`](#tidb_decode_sql_digests) | The `TIDB_DECODE_SQL_DIGESTS()` function is used to query the normalized SQL statements (a form without formats and arguments) corresponding to the set of SQL digests in the cluster. |
 | `VITESS_HASH(str)` | The `VITESS_HASH` function returns the hash of a string that is compatible with Vitess' `HASH` function. This is intended to help the data migration from Vitess. |
+| `TIDB_SHARD()` | The `TIDB_SHARD` function can be used to create a SHARD INDEX to scatter the index hotspot. A SHARD INDEX is an expression index with a `TIDB_SHARD`function as the prefix.|
```

Comment:
If SHARD INDEX is natural language, shall we use the lower case "shard index"? Or wrap it with backticks.
```suggestion
| `TIDB_SHARD()` | The `TIDB_SHARD` function can be used to create a SHARD INDEX to scatter the index hotspot. A SHARD INDEX is an expression index with a `TIDB_SHARD` function as the prefix.|
```